packages/server/graphql/rootMutation.ts

```diff
@@ -269,6 +273,8 @@ export default new GraphQLObjectType<any, Context>({
       pokerResetDimension,
       pokerAnnounceDeckHover,
       pokerSetFinalScore,
-      updateJiraDimensionField
+      updateJiraDimensionField,
+      messageAllSlackUsers,
```

Comment:
mutations should only live on 1 schema. can you remove these so they are just on the private one?